samples/document-snippets/family.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const Bigtable = require('@google-cloud/bigtable');
+const bigtable = new Bigtable();
+
+const snippets = {
+  createColmFamily: (instanceId, tableId, familyId, callback) => {
+    const instance = bigtable.instance(instanceId);
+    const table = instance.table(tableId);
+    const family = table.family(familyId);
+    // [START bigtable_create_family]
+    family
+      .create()
+      .then(result => {
+        const family = result[0];
+        // let apiResponse = result[1];
+      })
+      .catch(err => {
+        // handle error
+      });
+    // [END bigtable_create_family]
+    callback();
+  },
+  existsFamily: (instanceId, tableId, familyId, callback) => {
+    const instance = bigtable.instance(instanceId);
+    const table = instance.table(tableId);
+    const family = table.family(familyId);
+
+    // [START bigtable_exists_family]
+    family
+      .exists()
+      .then(result => {
+        const exists = result[0];
+      })
+      .catch(err => {
+        // handle error
+      });
+    // [END bigtable_exists_family]
+    callback();
+  },
+  getFamily: (instanceId, tableId, familyId, callback) => {
+    const instance = bigtable.instance(instanceId);
+    const table = instance.table(tableId);
+    const family = table.family(familyId);
+    // [START bigtable_get_family]
+    family
+      .get()
+      .then(result => {
+        const family = result[0];
+        // const apiResponse = result[1];
+      })
+      .catch(err => {
+        // handle error
```

Comment:
"Error handling omitted." or "Handle the error." would match our phrasing conventions better.